### PR TITLE
improve wording in skeleton doc

### PR DESCRIPTION
**Pull request by voroyam**
rephrase the wording to make it more clear

### CODE REVIEW

File: modules/admin_manual/pages/configuration/files/default_files_configuration.adoc [hunk 1]
```diff
@@ -20,7 +20,11 @@ image:skeleton-files1.png[image]
 == Additional Configuration
 
 The configuration option `skeletondirectory` available in your `config.php` allows you to configure the directory where the skeleton files are located.
+
 These files will be copied to the data directory of new users.
-Leave empty to not copy any skeleton files.
+
+Leave the `/core/skeleton` directory empty if you do not want to copy any skeleton files.
```
Comment (pako81) [inline modules/admin_manual/pages/configuration/files/default_files_configuration.adoc]: Just say "Leave this directory empty" - as you can theoretically also define a custom path apart from `/core/skeleton`

**Comment by pako81** [comment]
Also, this https://doc.owncloud.com/server/admin_manual/configuration/server/config_sample_php_parameters.html#the-directory-where-the-skeleton-files-are-located-these-files-will-be requires to be edited as well

File: modules/admin_manual/pages/configuration/files/default_files_configuration.adoc [hunk 2]
```diff
@@ -20,7 +20,11 @@ image:skeleton-files1.png[image]
...
+
+The value of the `skeletondirectory =>` key **must not be empty** if you decide to use it in your `config.php`.
```
Comment (settermjd) [inline modules/admin_manual/pages/configuration/files/default_files_configuration.adoc]: `skeletondirectory =>` can be simplified to just `skeletondirectory`.

File: modules/admin_manual/pages/configuration/files/default_files_configuration.adoc (hunk 1 above)
Comment (voroyam) [reply modules/admin_manual/pages/configuration/files/default_files_configuration.adoc]: changed

**Comment by voroyam** [comment]
> Also, this https://doc.owncloud.com/server/admin_manual/configuration/server/config_sample_php_parameters.html#the-directory-where-the-skeleton-files-are-located-these-files-will-be requires to be edited as well

can't do this. This script is copied from core and I have no idea how to change it.

My PR for merging the core and apps parameters was rotting as a PR for a year or so until I closed it.

File: modules/admin_manual/pages/configuration/files/default_files_configuration.adoc (hunk 2 above)
Comment (voroyam) [reply modules/admin_manual/pages/configuration/files/default_files_configuration.adoc]: changed

**Pull request merged**